src/main/resources/hudson/plugins/cobertura/CoberturaPublisher/config.jelly

```diff
@@ -5,6 +5,11 @@
              description="${%xml.report.pattern.description}">
         <f:textbox name="cobertura.coberturaReportFile" value="${instance.coberturaReportFile}"/>
     </f:entry>
+
+    <f:entry title="Enable New API">
```

Comment:
I would rather expect it to be implicit at some point

Reply:
You mean move it into advanced option?